service/pom.xml

```diff
@@ -17,6 +16,64 @@
             <version>${tds-common.version}</version>
         </dependency>
 
+        <!-- Existing TDS dependencies -->
+        <dependency>
```

Comment:
I tried to find everything being pulled in by the shared projects and excluded it.  Noting because if we see version clashes in the future we may need to revisit.